Users/coafile.rst

```diff
@@ -39,6 +39,16 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+``default_coafile`` is a normal coala configuration
+file with system wide effect.
```

Comment:
This is more or less mentioned... hm could you try to integrate your stuff better into the existing text here? You can also modify the old text if you think explanations are missing 👍

Reply:
I will just remove the highlighted lines.